azure/v12.0.0/README.md

```diff
@@ -0,0 +1,74 @@
+## :zap:  Giant Swarm Release 12.0.0 for Azure :zap:
+
+This is the first release to support kubernetes 1.17 on Azure.
+
+As of this release NGINX Ingress Controller App is optional and not pre-installed component on Azure.
+Upgrading existing clusters with pre-installed NGINX will leave NGINX unchanged.
+This allows NGINX App installations to be managed independently from the base platform lifecycle.
+It also enables use of other ingress controller alternatives without wasting resources where NGINX is not the preferred option.
```

Comment:
```suggestion
As of this release NGINX Ingress Controller App is optional and not pre-installed component on Azure.
This allows NGINX App installations to be managed independently from the base platform lifecycle. It is both benefit but also new responsibility to keep NGINX App installations up-to-date separately from rest of the cluster.
Making NGINX optional enables use of other ingress controller alternatives without wasting resources where NGINX is not the preferred option.
Upgrading existing tenant clusters with pre-installed NGINX will leave NGINX unchanged. Existing NGINX App custom resources will still have `giantswarm.io/managed-by: cluster-operator` label, but it will be ignored. The label will be cleaned up at a later point after all tenant clusters have been upgraded and Azure platform releases older than v12.0.0 archived.
```

@whites11 here I update the release notes to mention label will remain for some time; I've also created a TODO for Halo https://github.com/giantswarm/giantswarm/issues/12247 to perform the label and cluster-operator logic cleanup eventually https://github.com/giantswarm/giantswarm/issues/12247